src/assets/js/alma-widgets-inject.js

```diff
@@ -4,54 +4,80 @@
  * @package Alma_WooCommerce_Gateway
  */
 
-jQuery( document ).ready(
-	function($) {
-		var paymentPlansContainerId = "#alma-payment-plans";
-		var settings                = $( paymentPlansContainerId ).data( 'settings' )
-		if ( ! settings ) {
-			return;
-		}
-		var jqueryUpdateEvent = settings.jqueryUpdateEvent;
-		var firstRender       = settings.firstRender;
-
-		function isVisible( elem ) {
+/**
+ * Alma admin General Helpers.
+ */
+function AlmaWidgetHelper() {
+	return {
+		/**
+		 * Check if widget is visible.
+		 *
+		 * @param elem
+		 * @returns {boolean}
+		 */
+		isVisible: function( elem ) {
 			return ! ! ( elem && ( elem.offsetWidth || elem.offsetHeight || elem.getClientRects().length ) );
-		}
-
-		function getAmountElement() {
-			var amountQuerySelector = settings.amountQuerySelector;
-
+		},
+		/**
+		 * Get amount Element.
+		 *
+		 * @returns {null|*}
+		 */
+		getAmountElement: function() {
+			var paymentPlansContainerId = "#alma-payment-plans";
+			var settings                = jQuery( paymentPlansContainerId ).data( 'settings' );
+			var amountQuerySelector     = settings.amountQuerySelector;
 			if ( amountQuerySelector ) {
 				return document.querySelector( amountQuerySelector );
 			}
-
-			return null
-		}
-
-		window.AlmaInitWidget = function () {
+			return null;
+		},
+		/**
+		 * Get PaymentPlansContainerId.
+		 *
+		 * @returns String
+		 */
+		getPaymentPlansContainerId: function() {
+			return '#alma-payment-plans';
+		},
+		/**
+		 * Get settings.
+		 *
+		 * @returns {null|*}
+		 */
+		getSettings: function() {
+			var paymentPlansContainerId = "#alma-payment-plans";
+			return jQuery( paymentPlansContainerId ).data( 'settings' );
+		},
+		/**
+		 * AlmaInitWidget.
+		 *
+		 * @returns Null
+		 */
+		AlmaInitWidget: function () {
 			// Make sure settings are up-to-date after a potential cart_totals refresh.
-			var settings = $( paymentPlansContainerId ).data( 'settings' )
+			var settings = AlmaWidgetHelper().getSettings();
 			if ( ! settings ) {
-				return;
+				return null;
 			}
 
 			if (settings.hasExcludedProducts) {
-				return;
+				return null;
```

Comment:
This method should return nothing ... returning null in some case is not a consistent behavior.
```suggestion
				return;
```

Reply:
ok